tmk_core/common/action.c

```diff
@@ -357,6 +357,8 @@ void process_action(keyrecord_t *record, action_t action) {
                             dprint("MODS_TAP: Tap: unregister_code\n");
                             if (action.layer_tap.code == KC_CAPS) {
                                 wait_ms(TAP_HOLD_CAPS_DELAY);
+                            } else {
+                                wait_ms(TAP_CODE_DELAY);
```

Comment:
It may be worth doing this:
```suggestion
#if TAP_CODE_DELAY > 0
                            } else {
                                wait_ms(TAP_CODE_DELAY);
#endif
```

Reply:
As I understand it `wait_ms(0)` gets compiled out, but I could be wrong.